src/app/pages/deposits/deposits.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="body">
+  <h3 class="title">Deposits</h3>
+  <div class="row">
+    <div class="col-md-8">
+      <div class="container">
+        <div class="row">
+          <div class="col fromDate">
+            <input
+              [owlDateTime]="dt1"
+              [owlDateTimeTrigger]="dt1"
+              placeholder="Select Start Date"
+              class="form-control"
+              [(ngModel)]="startDateVal"
+            />
+            <owl-date-time [pickerType]="'calendar'" #dt1></owl-date-time>
+          </div>
+          <div class="col toDate">
+            <input
```

Comment:
Hey @Genza999, to improve usability for mouse users, we could consider adding labels to these input fields.

Reply:
Hey, worked on this @CryceTruly